<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salida de ejecuccion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnCerrar">
     <property name="text">
      <string>Cerrar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/48px-System-log-out.png</normaloff>../iconos/48px-System-log-out.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnLimpiar">
     <property name="text">
      <string>Limpiar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/48px-Edit-clear.png</normaloff>../iconos/48px-Edit-clear.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
